shared/types/field/dateField.ts

```diff
@@ -16,4 +16,5 @@ export type DateValidationOptions = {
 export interface DateFieldBase extends MyInfoableFieldBase {
   fieldType: BasicField.Date
   dateValidation: DateValidationOptions
+  invalidDaysOfTheWeek?: number[]
```

Comment:
should we use an enum as discussed in a previous PR?

Reply:
Yes. Updated in latest commit. Forget about that because I changed it in the other PR.